src/test/java/mil/dds/anet/test/resources/MergePeopleTest.java

```diff
@@ -0,0 +1,708 @@
+    // one without position full endTime(loser created),
+    // one with position full endTime (loser pos1 assigned)
+    // one without position full endTime (loser pos1 removed)
+    // one with position full endTime(loser pos2 assigned)
+    // one without position full endTime (loser pos2 removed)
+    // one with position full endTime(loser pos3 assigned)
+    // one without position full endTime (loser pos3 removed)
+    // one with position full endTime(loser pos4 assigned)
+    // one without position full endTime (loser pos4 removed)
+    // one with position full endTime(loser pos5 assigned)
+    // one without position full endTime (loser pos5 removed)
+    // one with position full endTime(loser pos6 assigned)
+    // one without position empty endTime (loser pos6 removed) -------------- endTime = Null
+    assertThat(personLoser.getPreviousPositions().size()).isEqualTo(13);
+
+    Person winner = mergePeople(personLoser.getUuid(), personWinner.getUuid(), false);
+    // There must be fourteen position histories,
+    // one without position full endTime(loser created),
+    // one with position full endTime (loser pos1 assigned)
+    // one without position full endTime (loser pos1 removed)
+    // one with position full endTime(loser pos2 assigned)
+    // one without position full endTime (loser pos2 removed)
+    // one with position full endTime(loser pos3 assigned)
+    // one without position full endTime (loser pos3 removed)
+    // one with position full endTime(loser pos4 assigned)
+    // one without position full endTime (loser pos4 removed)
+    // one with position full endTime(loser pos5 assigned)
+    // one without position full endTime (loser pos5 removed)
+    // one with position full endTime(loser pos6 assigned)
+    // one without position full endTime (loser pos6 removed)
+    // one without position empty endTime (winner created) -------------- endTime = Null
+    assertThat(
+        winner.getPreviousPositions().stream().filter(f -> Objects.isNull(f.getEndTime())).count())
+            .isEqualTo(1);
+    assertThat(winner.getPreviousPositions().stream()
+        .filter(f -> Objects.isNull(f.getEndTime()) && Objects.isNull(f.getPositionUuid())).count())
+            .isEqualTo(1);
+    assertThat(winner.getPreviousPositions().size()).isEqualTo(14);
+  }
+
+  @Test
+  public void loserHasPositionWinnerHasNoPositionHasPositionHistoryCopyPositionNotSelected() {
+    Position positionLoser = createPosition("Loser Position One");
+    // Active peoplePosition without person must be created when the position is created.
+    assertThat(positionLoser.getPreviousPeople().stream()
+        .filter(f -> Objects.isNull(f.getEndTime())).count()).isEqualTo(1);
+    assertThat(positionLoser.getPreviousPeople().stream()
+        .filter(f -> Objects.isNull(f.getEndTime()) && Objects.isNull(f.getPersonUuid())).count())
+            .isEqualTo(1);
+
+    Position positionWinnerOne = createPosition("Winner Position One");
```

Comment:
Probably nicer to use `positionWinner1`, ` positionWinner2`.

Reply:
Variable names changed.